system/zlib/Make.defs

```diff
@@ -23,4 +23,8 @@
 
 ifneq ($(CONFIG_LIB_ZLIB),)
 CONFIGURED_APPS += $(APPDIR)/system/zlib
+
+CFLAGS +=  ${INCDIR_PREFIX}$(APPDIR)/system/zlib/zlib
```

Comment:
but many intetrnal header files will expose to the global search path, which may introduce the conflict.

Reply:
Nope, it does not work. @xiaoxiang781216 what would be your suggestions? Abandon this change?

Reply:
Let me try creating `include/zlib.h` wrapper at application layer. Maybe that would work.